### PR TITLE
Feature/lint

### PR DESCRIPTION
**Pull request by ZhengHe-MD**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "compile:postcss": "postcss  --use precss --use postcss-cssnext -d lib src/*.css",
     "compile:source": "babel src --out-dir lib",
     "prepublish": "npm-run-all clean compile",
+    "lint": "eslint src test --ext .js,.jsx",
```
Comment (jimexist) [inline package.json:12]: maybe `.{js,jsx}`

File: test/MDSearchBox.spec.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
```
Comment (jimexist) [inline test/MDSearchBox.spec.js:2]: prefer disable-next line but YMMV

File: package.json (hunk 1 above)
Comment (ZhengHe-MD) [inline package.json:12]: http://eslint.org/docs/user-guide/command-line-interface#ext

File: test/MDSearchBox.spec.js (hunk 2 above)
Comment (ZhengHe-MD) [inline test/MDSearchBox.spec.js:2]: 恩。disable-next-line更清晰一些

**Pull request merged**